webservice/dispatcher.php

```diff
@@ -24,12 +24,15 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
+use PrestaShop\PrestaShop\Adapter\ContainerBuilder;
+
 ob_start();
 
 require_once dirname(__FILE__).'/../config/config.inc.php';
 
 // Cart is needed for some requests
 Context::getContext()->cart = new Cart();
+Context::getContext()->container = ContainerBuilder::getContainer('webservice', _PS_MODE_DEV_);
```

Comment:
This property must be declared in Context

Reply:
:ok_hand: